src/stores/media/index.ts

```diff
@@ -378,16 +382,47 @@ export const useMediaStore = defineStore('media', {
     }) {
       let errorMessage
       if (axios.isAxiosError(error)) {
-        errorMessage =
-          error.response?.status === 500
-            ? 'There was a problem with our servers'
-            : `Request failed with status ${
-                error.response?.status ?? 'unknown'
-              }`
+        // If the error is an axios error:
+        // If the error has a response property, and recieved a response that is not in the 2xx range,
+        // then the error is logged to Sentry
+        if (error.response) {
+          errorMessage = `Error fetching ${mediaType} from API. Request failed with status code: ${error.response.status}`
+          this.$nuxt.$sentry.captureEvent({
+            message: errorMessage,
+            extra: {
+              error,
+            },
+          })
+        } else if (error.request) {
+          // If the error has a request property, but no response, then we capture the event in Sentry
+          errorMessage = `Error fetching ${mediaType} from API. No response received from the server`
+          this.$nuxt.$sentry.captureEvent({
+            message: errorMessage,
+            extra: {
+              error,
+            },
+          })
+        } else {
+          // Something happened in setting up the request that triggered an Error
+          errorMessage = `Error fetching ${mediaType} from API. Unknown Axios error`
+          this.$nuxt.$sentry.captureEvent({
+            message: errorMessage,
+            extra: {
+              error,
+            },
+          })
```

Comment:
We can move the `sentry.captureEvent` calls to line 425, before `this._updateFetchState`, and only call once.

Reply:
Is there any scenario where it would get called twice? If you are refering on the times it appears in the code, i believe having a separate call for each exception allows us to fine-tune the amount of detail we want/need for each type of error. If you think that isn't needed, i'll change it

Reply:
Sorry, I wasn't clear :)
Currently, the same code is repeated 3 times:
```
          this.$nuxt.$sentry.captureEvent({
            message: errorMessage,
            extra: {
              error,
            },
          })
```
Since only the errorMessage changes, we can set the errorMessage inside the `if ..else`, and then call `sentry.captureEvent` outside of the `if ... else`.